js/Gameboard.js

```diff
@@ -1,23 +1,52 @@
-class Gameboard {
+  class Gameboard {
+    constructor() {
+      this.possibleCategories = this.generateCategories();
+    }
+
+  
+    const generateRandomIndex = (length) => {    
```

Comment:
You don't want to be creating named arrow functions inside of a class -- you should be creating methods instead like so:

```js
generateRandomIndex() {
  // do stuff
},

chooseQuestion() {
  // do stuff
}
```